vite.config.ts

```diff
@@ -2,7 +2,7 @@ import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
 
 export default defineConfig({
-  base: '/',
+  base: '/ui/',
```

Comment:
Explicitly setting the base here prevents the interface from being served on a different base path. Currently the router will take any base path like `/ui/metrics` or `/foo/bar/metrics``.

Reply:
Ah gotcha, let's not do that then :-)